scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -63,13 +63,18 @@ object Common {
 
     FileInputFormat
       .getInputPaths(conf)
-      .map { path =>
+      .flatMap { path =>
```

Comment:
I actually rethinking this and use `Try`, because in many ways it can throw IOException